lib/client.js

```diff
@@ -68,6 +68,10 @@ class Freshdesk {
 		makeRequest('GET', this._auth, `${this.baseUrl}/api/v2/tickets`, params, null, cb)
 	}
 
+	filterTickets (filter, cb) {
```

Comment:
The name `filter` is not appropriate for this parameter. You should use `query`, because in other methods `filter` is an object, and here `query` - if a formatted string.